cla_backend/apps/cla_eventlog/management/commands/find_and_delete_old_cases.py

```diff
@@ -22,7 +22,18 @@ def handle(self, *args, **kwargs):
         instance = FindAndDeleteCasesUsingCreationTime()
         cases = instance.get_eligible_cases()
         if len(args) and args[0] == "delete":
-            instance.run()
+            if len(args) > 1 and args[1] == "no-input":
+                instance.run()
+            elif sys.argv[1] == "test":
```

Comment:
This is hard to notice (that it's different from args). I would consider making `sys.argv[1]` into a variable with a clear name a bit higher up and then we can use it here without confusion